lib/ansible/modules/network/nxos/nxos_ospf_vrf.py

```diff
@@ -405,14 +405,15 @@ def main():
     if state == 'absent' and existing:
         state_absent(module, existing, proposed, candidate)
 
-    if candidate:
+    if not module.check_mode and candidate:
```

Comment:
Thanks for your contribution.
Please update this code block to the following to avoid repetitive code:
```
if candidate:
    candidate = candidate.items_text()
    result['commands'] = candidate
    if not module.check_mode:
        load_config(module, candidate)
        result['changed'] = True
```